### PR TITLE
Qpqdata

### PR DESCRIPTION
**Pull request by marijana777**

### CODE REVIEW

File: pyigm/cgm/qpq.py [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+    basestring = str
+
+
+def load_qpq(v):
+    """ Reads table with data
+
+    Parameters
+    ----------
+    v : int
+      dataset (6,7,8)
```
Comment (profxj) [inline pyigm/cgm/qpq.py]: Add 5

**Comment by profxj** [comment]
This is a fine start @marijana777 

Now I request that you push the data into a
CGMAbsSurvey object.  And I have pulled in
the add_richter17 branch to make sure you 
have the latest code on that..

Am closing this PR for now.

**Comment by marijana777** [comment]
QPQ data are now in CGMAbsSurvey object(s)

File: pyigm/surveys/llssurvey.py [hunk 2]
```diff
@@ -250,6 +250,7 @@ def load_HDLLS(cls, load_sys=True, grab_spectra=False, isys_path=None):
 
         # Set data path and metallicity
         spath = pyigm_path+"/data/LLS/HD-LLS/Spectra/"
+        lls_survey.build_all_abs_sys()
```
Comment (profxj) [inline pyigm/surveys/llssurvey.py]: did you add this @marijana777 ?

I don't think we want this here.

**Review by profxj — CHANGES_REQUESTED** [review]
want to make sure everything works before merging.

File: pyigm/surveys/llssurvey.py (hunk 2 above)
Comment (marijana777) [reply pyigm/surveys/llssurvey.py]: I did not, as I know

File: pyigm/cgm/qpq.py (hunk 1 above)
Comment (marijana777) [reply pyigm/cgm/qpq.py]: Completed

**Comment by profxj** [comment]
The test_qpq() is failing on Travis.
https://travis-ci.org/pyigm/pyigm/jobs/369629383

Please fix it.

**Comment by marijana777** [comment]
test_qpq.py  takes more than 10min to load QPQ6,7,8.  I could decrease this time (by not loading full samples, or all QPQ6,7,8) or look how to use travis_wait

**Comment by marijana777** [comment]
Loading QPQ 6,7,8 takes 3.25, 8.7, 2.25 min. Changed test_qpq.py such that now it takes 4min9sec, and is loading full sample only for QPQ6. Would this be fine for Travis?

**Comment by marijana777** [comment]
I think that Travis was failing because test_qpq.py was taking more than 10 min, and that this is fixed now.

**Review by profxj — APPROVED** [review]
Thanks for all of this Marijana.

**Comment by profxj** [comment]
merging.

**Pull request merged**